launch.go

```diff
@@ -536,6 +538,7 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 		"SD_CACHE_MD5CHECK":       fmt.Sprintf("%v", cacheMd5Check),
 		"SD_CACHE_MAX_SIZE_MB":    fmt.Sprintf("%v", cacheMaxSizeInMB),
 		"SD_CACHE_MAX_GO_THREADS": fmt.Sprintf("%v", cacheMaxGoThreads),
+		"SD_SCHEDULING_BUILD":     isScheduler,
```

Comment:
maybe `SD_SCHEDULED_BUILD` or 
not sure if it would be better if we could have something like `SD_TRIGGER` or `SD_START_FROM` and it would be `scheduler` or `commit` or `manual`

Reply:
Fix env name to `SD_SCHEDULED_BUILD`.